src/test/java/com/wikia/webdriver/testcases/mercurytests/RedirectionTests.java

```diff
@@ -36,4 +44,45 @@ public void RedirectionTest_001_RedirectFromWWW() {
     new ArticlePageObject(driver).openWikiPage(url);
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
   }
+
+  private void redirectFromFullSiteToOasis() {
+    MercuryFooterComponentObject footer = new MercuryFooterComponentObject(driver);
+    ArticleNavigationComponentObject navigation = new ArticleNavigationComponentObject(driver);
+    SkinHelper helper = new SkinHelper(driver);
+
+    new ArticlePageObject(driver).openWikiPage(url);
+    footer.clickFullSiteLink();
+    navigation.clickRandomArticle();
+
+    Assertion.assertTrue(helper.isSkin(Skin.OASIS));
+  }
+
+  @Test(groups = "RedirectionTest_002")
+  @Execute(onWikia = "mercuryautomationtesting")
+  public void RedirectionTest_002_RedirectFromFullSiteToOasis() {
+    redirectFromFullSiteToOasis();
+  }
+
+  @Test(groups = "RedirectionTest_003")
+  @Execute(onWikia = "mercuryautomationtesting", asUser = User.USER)
```

Comment:
Ok, then please rename methods to:
- RedirectionTest_002_RedirectFromFullSiteToOasis --> RedirectionTest_002_RedirectFromFullSiteToOasisForAnons
- RedirectionTest_003_RedirectFromFullSiteToOasis --> RedirectionTest_003_RedirectFromFullSiteToOasisForLoggedInUsers
or something like that